packages/yarnpkg-fslib/sources/NodeFS.ts

```diff
@@ -284,14 +284,20 @@ export class NodeFS extends BasePortableFakeFS {
     return this.realFs.mkdirSync(npath.fromPortablePath(p), opts);
   }
 
-  async rmdirPromise(p: PortablePath) {
+  async rmdirPromise(p: PortablePath, opts?: RmdirOptions) {
     return await new Promise<void>((resolve, reject) => {
-      this.realFs.rmdir(npath.fromPortablePath(p), this.makeCallback(resolve, reject));
+      // TODO: always pass opts when min node version is 12.10+
+      if (opts) {
+        this.realFs.rmdir(npath.fromPortablePath(p), opts, this.makeCallback(resolve, reject));
+      }
+      else {
+        this.realFs.rmdir(npath.fromPortablePath(p), this.makeCallback(resolve, reject));
+      }
```

Comment:
Can you `1tbt`-ify this? (That's the brace style we use in the rest of the codebase, would probably be a good idea to enforce it with an ESLint rule)

Reply:
You mean put the else on the same line?

Reply:
Yep, the else should be on the same line as the closing if brace.

Reply:
ok cool oddly the `1tbt` turned up no results when I searched it.

Reply:
`1tbt` is an acronym for `one true brace style` - https://eslint.org/docs/rules/brace-style